src/components/footer/index.tsx

```diff
@@ -0,0 +1,109 @@
+            <FooterLogo>
+              <img src={logo} />
+            </FooterLogo>
+            <FooterText>Some text will be here two - three lines</FooterText>
+            <FooterSocials>
+              <FooterIcon>
+                <FooterLink to="#">
+                  <InstagramIcon />
+                </FooterLink>
+              </FooterIcon>
+              <FooterIcon>
+                <FooterLink to="#">
+                  <FacebookIcon />
+                </FooterLink>
+              </FooterIcon>
+              <FooterIcon>
+                <FooterLink to="#">
+                  <EmailIcon />
+                </FooterLink>
+              </FooterIcon>
+            </FooterSocials>
+          </FooterLeft>
+          <FooterRight>
+            <FooterMenu>
+              <FooterTitle>Shop</FooterTitle>
+              <FooterMenuList>
+                <FooterMenuItem>
+                  <FooterLink to="#">Login</FooterLink>
+                </FooterMenuItem>
+                <FooterMenuItem>
+                  <FooterLink to="#">Wishlist</FooterLink>
+                </FooterMenuItem>
+                <FooterMenuItem>
+                  <FooterLink to="#">Cart</FooterLink>
+                </FooterMenuItem>
+              </FooterMenuList>
+            </FooterMenu>
+            <FooterMenu>
+              <FooterTitle>Company</FooterTitle>
+              <FooterMenuList>
+                <FooterMenuItem>
+                  <FooterLink to="#">About US</FooterLink>
+                </FooterMenuItem>
+                <FooterMenuItem>
+                  <FooterLink to="#">Terms & Conditions</FooterLink>
+                </FooterMenuItem>
+                <FooterMenuItem>
+                  <FooterLink to="#">Privacy Policy</FooterLink>
+                </FooterMenuItem>
+                <FooterMenuItem>
+                  <FooterLink to="#">Contact Us</FooterLink>
+                </FooterMenuItem>
+              </FooterMenuList>
+            </FooterMenu>
+            <FooterMenu>
+              <FooterTitle>Help</FooterTitle>
+              <FooterMenuList>
+                <FooterMenuItem>
+                  <FooterLink to="#">FAQs</FooterLink>
+                </FooterMenuItem>
+                <FooterMenuItem>
+                  <FooterLink to="#">Wishlist</FooterLink>
+                </FooterMenuItem>
+                <FooterMenuItem>
+                  <FooterLink to="#">Shipping and Delivery</FooterLink>
+                </FooterMenuItem>
+                <FooterMenuItem>
+                  <FooterLink to="#">Rent your closet</FooterLink>
+                </FooterMenuItem>
+                <FooterMenuItem>
+                  <FooterLink to="#">Resell your closet</FooterLink>
+                </FooterMenuItem>
+              </FooterMenuList>
+            </FooterMenu>
+          </FooterRight>
+        </FooterInner>
+        <Credentials>© 2023 Black Cycle</Credentials>
+      </Container>
+    </StyledFooter>
+  );
+};
```

Comment:
1.3.3. PR have checklist, pleae check it before send things to review

Reply:
fixed in [91dd88a](https://github.com/ZenBit-Tech/faq_team_fe/pull/8/commits/91dd88a27d00755183ac4283d923f605ff2c23df)